docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_pageindicator.htm

```diff
@@ -65,18 +65,20 @@ <h2><a id='manual-constructor'></a>How to Create page indicator</h2>
 
```

Comment:
Question. Don't we need `<p>HTML code:</p>` here?

Reply:
Yes, we need. I will fix it.

Reply:
Fixed